data/geany-3.0.css

```diff
@@ -1,10 +0,0 @@
-@import "geany.css";
-
-/* make close button on the editor's tabs smaller */
-#geany-close-tab-button {
-	-GtkWidget-focus-padding: 0;
-	-GtkWidget-focus-line-width: 0;
-	-GtkButton-default-border: 0;
-	-GtkButton-default-outside-border: 0;
-	-GtkButton-inner-border: 0;
-}
```

Comment:
It's not really a loss of flexibility, I just saw the opportunity to remove one redundant file. If something else comes up, we can add other CSS files and load them conditionally based on version, like is done for 3.20. The main improvement is that it's now using the Cascading effect instead of relying on `@import`ing another file.